src/test/unit/algorithms/SpatialPoolerTest.cpp

```diff
@@ -2018,6 +2019,7 @@ namespace nupic {
     }
     NTA_CHECK(check_vector_eq(trueNeighborsMap5, neighborsMap));
 
+    numColumns = 8;
```

Comment:
After the three tests at 30 columns (3, 4, and 5), back to 8 columns for remaining tests here (6 - 10).